cmd/lncli/commands.go

```diff
@@ -91,8 +91,7 @@ var newAddressCommand = cli.Command{
 	Description: `
 	Generate a wallet new address. Address-types has to be one of:
 	    - p2wkh:  Push to witness key hash
-	    - np2wkh: Push to nested witness key hash
-	    - p2pkh:  Push to public key hash (can't be used to fund channels)`,
+	    - np2wkh: Push to nested witness key hash`,
```

Comment:
Didn't notice this before, but is this really "push to" not "pay to"?

Reply:
lol yeh dunno what happened there, fixed!